doc/source/reference/random/generator.rst

```diff
@@ -0,0 +1,82 @@
+.. currentmodule:: numpy.random
+
+Random Generator
+----------------
+The `~Generator` provides access to
+a wide range of distributions, and served as a replacement for
+:class:`~numpy.random.RandomState`.  The main difference between
+the two is that ``Generator`` relies on an additional BitGenerator to
+manage state and generate the random bits, which are then transformed into
+random values from useful distributions. The default BitGenerator used by
+``Generator`` is :class:`~xoroshiro128.Xoroshiro128`.  The BitGenerator
+can be changed by passing an instantized BitGenerator to ``Generator``.
+
+
+.. autoclass:: Generator
+	:exclude-members:
+
+Accessing the BitGenerator
+==========================
+.. autosummary::
+   :toctree: generated/
+
+   ~Generator.bit_generator
+
+Simple random data
+==================
+.. autosummary::
+   :toctree: generated/
+
+   ~Generator.integers
+   ~Generator.random_sample
```

Comment:
At the developer meeting, I think we agree to swap out `random_sample` in favor of just `Generator.random`?

(`random_sample` potentially confusing with `random.sample()` from Python's `random` module, which does something different)

Reply:
that is still a TODO item on the [checklist](https://github.com/mattip/numpy/issues/18#issuecomment-491937089)